keystone_performance/load_test.py

```diff
@@ -4,6 +4,8 @@
 import itertools
 import json
 import time
+import random
+import requests
```

Comment:
"import requests" belongs in the next section since it's not part of the standard library.